tests/e2e_logging.py

```diff
@@ -593,6 +611,8 @@ def run(args):
         network = test_view_history(network, args)
         network = test_primary(network, args)
         network = test_metrics(network, args)
+        if args.enclave_type != "virtual":
```

Comment:
Why do we skip this in virtual? I think its fine if the endpoint returns `{0, 0, 0}` for a virtual enclave, but we can still test it returns something of the correct shape?